console/src/pages/organizations/OrganizationUsersTab.tsx

```diff
@@ -19,22 +23,50 @@ import {
 import { Link } from 'react-router-dom';
 import { DateTime } from 'luxon';
 import { timestampDate } from '@bufbuild/protobuf/wkt';
-import { useParams } from 'react-router';
+import { useNavigate, useParams } from 'react-router';
 import { Badge } from '@/components/ui/badge';
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from '@/components/ui/alert-dialog';
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormDescription,
+  FormField,
+  FormItem,
+  FormLabel,
+} from '@/components/ui/form';
+import { z } from 'zod';
+import { useForm } from 'react-hook-form';
+import { AlertDialogCancel } from '@radix-ui/react-alert-dialog';
+import { Input } from '@/components/ui/input';
+import { Switch } from '@/components/ui/switch';
+import { CirclePlus } from 'lucide-react';
+import { toast } from 'sonner';
+import { User } from '@/gen/tesseral/backend/v1/models_pb';
 
 export const OrganizationUsersTab = () => {
   const { organizationId } = useParams();
-  const { data: listUsersResponse } = useQuery(listUsers, {
+  const { data: listUsersResponse, refetch } = useQuery(listUsers, {
     organizationId,
   });
 
   return (
     <Card>
-      <CardHeader>
-        <CardTitle>Users</CardTitle>
-        <CardDescription>
-          A user is what people using your product log into.
-        </CardDescription>
+      <CardHeader className="flex flex-row items-center justify-between space-x-4">
+        <div>
+          <CardTitle>Users</CardTitle>
+          <CardDescription>
+            A user is what people using your product log into.
+          </CardDescription>
+        </div>
+        <CreateUserButton />
```

Comment:
Can you make this follow this style?

https://github.com/tesseral-labs/tesseral/blob/698607eabaaf6e1cb2c2f1386e2aefcbcbca07eb/console/src/pages/api-keys/ListAPIKeysPage.tsx#L97-L108

We'll make a component for the pattern, and fix the spacing issues on small screens as part of that component, but for now being consistent will make a later grep easier.